Applications/DataExplorer/DataView/MeshElementRemovalDialog.cpp

```diff
@@ -78,13 +77,21 @@ void MeshElementRemovalDialog::accept()
 	}
 	if (this->boundingBoxCheckBox->isChecked())
 	{
-		const MeshLib::Node min(this->xMinEdit->text().toDouble(),
-		                        this->yMinEdit->text().toDouble(),
-		                        this->zMinEdit->text().toDouble(),0);
-		const MeshLib::Node max(this->xMaxEdit->text().toDouble(),
-		                        this->yMaxEdit->text().toDouble(),
-		                        this->zMaxEdit->text().toDouble(),0);
-		ex.searchByBoundingBox(min, max);
+		std::vector<MeshLib::Node*> const& nodes (_project.getMesh(this->meshNameComboBox->currentText().toStdString())->getNodes());
+		GeoLib::AABB<MeshLib::Node> aabb(nodes.begin(), nodes.end());
+		MeshLib::Node minAABB = aabb.getMinPoint();
+		MeshLib::Node maxAABB = aabb.getMaxPoint();
+		double const eps (std::numeric_limits<double>::epsilon());
+
+		// only extract bounding box parameters that have been edited (otherwise there will be rounding errors!)
+		minAABB[0] = (aabb_edits[0]) ? this->xMinEdit->text().toDouble() : (minAABB[0] - eps);
```

Comment:
Can it happen, that  this function is invoked multiple times? If so, then the bounding box will grow every time by an eps.
I was wondering if this is the right location to make the bounding box larger.
The method [`AABB::containsPoint()`](http://doxygen.opengeosys.org/d6/d5d/classGeoLib_1_1AABB.html#ac4478094dbf00a1dd5de6f5d27f8d4db) checks for point `p` if `min <= p <= max` for all x, y, z, so it includes the boundary points already.